nomenclature/definitions/variable/technology/electricity-grid.yaml

```diff
@@ -1,25 +1,90 @@
 # List of variables related to the electricity grid
+# description of variables related to electricity demand
+Value of Lost Load|Electricity:
+   description: Cost of demand-not-served. Cost of load curtailment. Value of Lost Load (VoLL)
+   unit: €/MWh
 
-Network|Electricity|Demand|Reserve|Frequency Containment:
-   description: demand in frequency containment reserve (FCR)            
+Demand not served|Electricity:
+   description: Demand-not-served.
+   unit: MWh
+
+Reserve|Electricity|Requirement|Frequency Containment:
+   description: Demand in frequency containment reserve (FCR)
    unit: MW
-   
-Network|Electricity|Demand|Reserve|Automatic Frequency Restoration:
-   description: demand in automatic frequency restoration reserve (AFRR)             
+
+Reserve|Electricity|Requirement|Automatic Frequency Restoration:
+   description: Demand in automatic frequency restoration reserve (AFRR)
    unit: MW
-   
-Network|Electricity|Demand|Inertia:
-   description: demand in inertia            
+
+Network|Electricity|Requirement|Inertia:
+   description: Demand in inertia
    unit: second
-   
-Maximum Flow|Electricity|Grid:
-   description: maximum flow of an interconnection between 2 regions
-   unit: MW
-   
-Minimum Flow|Electricity|Grid:
-   description: minimum flow of an interconnection between 2 regions
+
+# description of variables related to transmission lines
+Network|Electricity|Active Power Flow:
+   description: Active power flow through a line
    unit: MW
-   
-Impedance|Electricity|Grid:
-   description: impedance of an interconnection between 2 regions
+
+Network|Electricity|Investment:
+   description: Investment decision. Note that it is a output data from model and if it makes a product with the investmen cost of the line, we get the investment in monetary unit.
+   unit: [Yes, No]
+
+Network|Electricity|Impedance:
+   description: Impedance of an interconnection between 2 regions
    unit: S
+
+Network|Electricity|Investment:
+   description: Overnight investment (capital) cost in transmission lines of the power network
+   unit: billion US$2010/yr
+
+Network|Electricity|Fixed Charge Rate:
```

Comment:
When I hear "Rate", I think of a cost (or payment by a consumer). I find it confusing that this is then described as a percentage. In my opinion, this should be either given as a cost for sending a quantity over the line, or the variable should be named "Ratio" (maybe)?

Reply:
This term is the usual convention for that purpose
See for example
https://www.e-education.psu.edu/eme801/node/560

Reply:
> This term is the usual convention for that purpose
> See for example
> https://www.e-education.psu.edu/eme801/node/560

I agree too with the term "Fixed Charge Rate". 
@openENTRANCE/coordination , any opinion?

Reply:
also agree